api/src/main/java/io/strimzi/api/kafka/model/balancing/KafkaRebalanceState.java

```diff
@@ -34,7 +36,7 @@
      *     <dt>ProposalReady</dt><dd>When the user sets annotation strimzi.io/rebalance=refresh and the proposal is already ready.</dd>
      * </dl>
      */
-    ProposalReady,
+    ProposalReady(List.of(KafkaRebalanceAnnotation.approve, KafkaRebalanceAnnotation.refresh, KafkaRebalanceAnnotation.none)),
```

Comment:
Why is `none` an allowed annotation? It doesn't serve any purpose? I thought you had explicit logic to handle the (very unlikely case) the user actually put `annotation=none`?

Reply:
Sorry @tomncooper. Looks like I forgot to stash these enum changes. Let me do it

Reply:
Removed